release_notes/topics/4_5_0_final.adoc

```diff
@@ -0,0 +1,22 @@
+= Signature SPI
+
+The Signature SPI makes it possible to plug-in additional signature algorithms. Not only does it enable
+additional signatures, but it also enables changing how signatures are generated. For example this would allow
+using an HSM device to sign tokens.
+
```

Comment:
The Signature SPI makes it possible to plug-in additional signature algorithms. This enables additional signatures and also enables changing how signatures are generated. For example, using this allows using an HSM device to sign tokens.